Windows does not build with Program Files installed SDK

<!--
Before filing a bug
- Ensure the bug reproduces on the latest commit of your release branch.
- Search existing issues and make sure this issue is not already filed.
-->

**Describe the bug**
Windows does not build with the latest commit in master. It fails to find the Azure Kinect SDK.

**To Reproduce**
Steps to reproduce the behavior:
1. Run "catkin_make" to try and build the project

Observe that a CMakeLists.txt error is produced saying that the SDK cannot be found.

**Expected behavior**
Project builds.

**Logs**
```
Finding K4A SDK binaries
-- Azure Kinect SDK Version File Contents: 1.1.0

-- PACKAGE_VERSION: 1.1.0
-- PACKAGE_FIND_VERSION: 1.1.0
CMake Error at Azure_Kinect_ROS_Driver/CMakeLists.txt:199 (message):
  K4A SDK not found in Program Files or ./ext/sdk.  Please install the Azure
  Kinect SDK.


-- Configuring incomplete, errors occurred!
```

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version: 1903

**Additional context**
Add any other context about the problem here.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -211,24 +211,22 @@ if (${K4A_INSTALL_NEEDED})
   set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS ${K4A_DLL_FILES})
 
   # We need to copy the DLLs into the CATKIN_PACKAGE_LIB_DESTINATION so
-  # the node executable can find them on launch
-  if (${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
-    set(DLL_COPY_DIRECTORY "${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_BIN_DESTINATION}")
-  elseif(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
-    set(DLL_COPY_DIRECTORY "${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_LIB_DESTINATION}")
-  endif()
+  # the node executable can find them on launch, and CATKIN_PACKAGE_BIN_DESTINATION
+  # so the nodelet can find them on launch
+  set(DLL_COPY_DIRECTORY "${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_BIN_DESTINATION};${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_LIB_DESTINATION}")
 
-  file(MAKE_DIRECTORY "${DLL_COPY_DIRECTORY}")
+  foreach(DIRECTORY ${DLL_COPY_DIRECTORY})
+    file(MAKE_DIRECTORY "${DIRECTORY}")
+  endforeach(DIRECTORY)
 
   foreach(DLL ${K4A_DLL_FILES})
-    file(COPY "${DLL}" DESTINATION "${DLL_COPY_DIRECTORY}")
-    
-    get_filename_component(DLL_NAME ${DLL} NAME)
-
-    message(STATUS "Copied dll from ${DLL_NAME} to ${DLL_COPY_DIRECTORY}")
-
-    # Tell cmake that we need to clean up these DLLs on a "make clean"
-    set_property(DIRECTORY APPEND PROPERTY ADDITIONAL_MAKE_CLEAN_FILES "${DLL_COPY_DIRECTORY}/${DLL_NAME}")
+    foreach(DIRECTORY ${DLL_COPY_DIRECTORY})
+      file(COPY "${DLL}" DESTINATION "${DIRECTORY}")
+      get_filename_component(DLL_NAME ${DLL} NAME)
+      message(STATUS "Copied dll from ${DLL_NAME} to ${DIRECTORY}")
+      # Tell cmake that we need to clean up these DLLs on a "make clean"
+      set_property(DIRECTORY APPEND PROPERTY ADDITIONAL_MAKE_CLEAN_FILES "${DIRECTORY}/${DLL_NAME}")
+    endforeach(DIRECTORY)
   endforeach(DLL)
   
 endif()

--- a/cmake/azure-kinect-sensor-sdk-config-version.cmake
+++ b/cmake/azure-kinect-sensor-sdk-config-version.cmake
@@ -7,7 +7,8 @@ find_file(versionfile "version.txt"
       NO_DEFAULT_PATH)
 
 if(NOT versionfile)
-  message(FATAL_ERROR "Azure Kinect SDK is not installed properly. It must be installed to the standard install location.")
+  message(STATUS "Azure Kinect SDK is not installed to Program Files.")
+  return()
 endif()
 
 set(versionfilecontents "0.0.0")

--- a/cmake/azure-kinect-sensor-sdk-install.cmake
+++ b/cmake/azure-kinect-sensor-sdk-install.cmake
@@ -2,11 +2,20 @@
 # Licensed under the MIT License.
 
 message("-- Custom K4A install -- ")
-file(MAKE_DIRECTORY "${DLL_COPY_DIRECTORY}")
-message("Installing to: ${DLL_COPY_DIRECTORY}")
+  # Tell cmake that we need to reconfigure if any of the DLL files change
+  set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS ${K4A_DLL_FILES})
 
-foreach(DLL ${K4A_DLL_FILES})
-    message("Copying DLL from ${DLL} to ${DLL_COPY_DIRECTORY}")
-    file(COPY "${DLL}" DESTINATION "${DLL_COPY_DIRECTORY}")
-endforeach(DLL)
+  foreach(DIRECTORY ${DLL_COPY_DIRECTORY})
+    file(MAKE_DIRECTORY "${DIRECTORY}")
+  endforeach(DIRECTORY)
+
+  foreach(DLL ${K4A_DLL_FILES})
+    foreach(DIRECTORY ${DLL_COPY_DIRECTORY})
+      file(COPY "${DLL}" DESTINATION "${DIRECTORY}")
+      get_filename_component(DLL_NAME ${DLL} NAME)
+      message(STATUS "Copied dll from ${DLL_NAME} to ${DIRECTORY}")
+      # Tell cmake that we need to clean up these DLLs on a "make clean"
+      set_property(DIRECTORY APPEND PROPERTY ADDITIONAL_MAKE_CLEAN_FILES "${DIRECTORY}/${DLL_NAME}")
+    endforeach(DIRECTORY)
+  endforeach(DLL)
 message("-- Custom K4A install finished -- ")

